entity-framework/core/querying/client-eval.md

```diff
@@ -43,7 +43,7 @@ public static string StandardizeUrl(string url)
 }
 ```
 
-## Disabling client evaluation
+## How to find code that causes client evaluation
```

Comment:
Not sure this is an improvement

Reply:
The idea is that when you configure the warning to throw you are effectively disabling client evaluation. And in general we talk about it that way, so having that in _a_ title should help, but I am ok with adding more titles.